src/qt/signverifymessagedialog.cpp

```diff
@@ -214,6 +207,27 @@ void SignVerifyMessageDialog::on_verifyMessageButton_VM_clicked()
             QString("<nobr>") + tr("Message verified.") + QString("</nobr>")
         );
         return;
+    case MessageVerificationResult::OK_TIMELOCKED:
+        ui->statusLabel_VM->setText(
+            QString("<nobr>") + tr("Message verified, but includes timelocks.") + QString("</nobr>")
+        );
+        return;
+    case MessageVerificationResult::INCONCLUSIVE:
+        ui->statusLabel_VM->setText(
+            QString("<nobr>") + tr("Inconclusive.") + QString("</nobr>")
+        );
+        return;
+    case MessageVerificationResult::ERR_INVALID:
+        ui->statusLabel_VM->setText(
+            tr("Some check failed.")
+        );
+        return;
+    case MessageVerificationResult::ERR_POF:
+        // TODO: support proof of funds verifications
+        ui->statusLabel_VM->setText(
+            tr("Proof of funds verification unavailable right now.")
```

Comment:
In 8f9c561c7b0887b98ce5a6670428fdeb1590deb1 "add basic BIP-322 message signing support"

Missing `QString("</nobr>"`.